puppet/modules/quickstack/manifests/neutron/compute.pp

```diff
@@ -127,6 +129,20 @@
       vxlan_udp_port      => $ovs_vxlan_udp_port,
       veth_mtu            => $veth_mtu,
     }
+
+    # check if opendaylight needs to be configured.
+    if ('opendaylight' in $ml2_mechanism_drivers) {
+      # OVS manager
```

Comment:
agreed, we're actually considering putting them in the odl puppet module itself.

the ipaddress should be abstracted to pass and interface name.